### PR TITLE
add aws-cli and its deps

### PR DESCRIPTION
**Pull request by tuananh**
Signed-off-by: Tuan Anh Tran <me@tuananh.org>

### CODE REVIEW

**Comment by tuananh** [comment]
im getting this issue when trying to build an image with this pkg

any pointer on why?

> Dec 28 13:57:03.126 [DEBUG] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:] ERROR: libffi-3.4.4-r0: trying to overwrite usr/share/info/dir owned by groff-1.22.3-r0.

**Comment by tuananh** [comment]
i just look at a bunch of py3 packages and it seems wolfi is accepting downloading .whl files? i thought that's built files and wolfi prefer building from src?

**Comment by tuananh** [comment]
i test by creating an image with apko. seems like all is good

image

**Review by dlorenc — DISMISSED** [review]

**Comment by tuananh** [comment]
weird, i can build py3-yaml locally? any idea why it would break

**Comment by tuananh** [comment]
```
2023/01/06 02:22:30 ERROR: failed to build package. the build environment has been preserved:
2023/01/06 02:22:30 melange (py3-asn1/x86_64):   workspace dir: /tmp/melange-workspace-270235846
2023/01/06 02:22:30 melange (py3-asn1/x86_64):   guest dir: /tmp/melange-guest-449109328
Error: failed to build package: unable to generate index: failed to parse package /home/runner/work/os/os/packages/x86_64/libxau-dev-1.0.10-r0.apk: expandApk(): expandApk error 5: expandApkWriter.Next error 5: open /tmp/734600351/stream-1.tar.gz: too many open files
2023/01/06 02:22:30 error during command execution: failed to build package: unable to generate index: failed to parse package /home/runner/work/os/os/packages/x86_64/libxau-dev-1.0.10-r0.apk: expandApk(): expandApk error 5: expandApkWriter.Next error 5: open /tmp/734600351/stream-1.tar.gz: too many open files
```

doesn't look related to this PR :-s

File: Makefile [hunk 1]
```diff
@@ -281,6 +281,23 @@ $(eval $(call build-package,cython,0.29.32-r0))
 $(eval $(call build-package,util-linux,2.38.1-r0))
 $(eval $(call build-package,glib,2.74.3-r0))
 $(eval $(call build-package,jansson,2.14-r0))
+$(eval $(call build-package,py3-asn1,0.4.8-r0))
+$(eval $(call build-package,py3-botocore,1.21.49-r0))
+$(eval $(call build-package,py3-docutils,0.19-r0))
+$(eval $(call build-package,py3-jmespath,0.10.0-r0))
+$(eval $(call build-package,py3-rsa,4.9-r0))
+$(eval $(call build-package,py3-s3transfer,0.5.1-r0))
+$(eval $(call build-package,cython,0.29.32-r0))
```
Comment (luhring) [inline Makefile:290]: There's already a line for cython

File: Makefile [hunk 2]
```diff
@@ -281,6 +281,23 @@ $(eval $(call build-package,cython,0.29.32-r0))
...
+$(eval $(call build-package,yaml,0.2.5-r0))
```
Comment (luhring) [inline Makefile:291]: There's already a line for yaml

File: Makefile [hunk 3]
```diff
@@ -281,6 +281,23 @@ $(eval $(call build-package,cython,0.29.32-r0))
...
+$(eval $(call build-package,py3-yaml,6.0-r0))
+$(eval $(call build-package,py3-dateutil,2.8.2-r0))
+$(eval $(call build-package,py3-urllib3,1.26.13-r0))
+$(eval $(call build-package,py3-hatchling,1.11.1-r0))
+$(eval $(call build-package,py3-pathspec,0.10.3-r0))
+$(eval $(call build-package,py3-pluggy,1.0.0-r0))
+$(eval $(call build-package,py3-colorama,0.4.6-r0))
+$(eval $(call build-package,groff,1.22.3-r0))
```
Comment (luhring) [inline Makefile:299]: There's already a line for groff

**Review by luhring — CHANGES_REQUESTED** [review]

**Review by luhring — APPROVED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**